db/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.0.2 on 2024-11-05 17:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Guild',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(null=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Race',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(blank=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Skill',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('bonus', models.CharField(max_length=255)),
+                ('race', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='skills', to='db.race')),
```

Comment:
The ForeignKey reference 'to='db.race'' should be 'to='app_name.Race'' where 'app_name' is the name of the app containing the models. Ensure that the correct app name is used to avoid migration errors.